server/db/schema/events.schema.ts

```diff
@@ -0,0 +1,34 @@
+import { relations } from 'drizzle-orm';
+import {
+  pgTable,
+  serial,
+  varchar,
+  timestamp,
+  integer,
+} from 'drizzle-orm/pg-core';
+
+import { clubs, persons } from '.';
+
+export const events = pgTable('events', {
+  id: serial('id').primaryKey(),
+  name: varchar('name', { length: 128 }).notNull(),
+  description: varchar('description', { length: 512 }),
+  startDate: timestamp('date').notNull(),
+  endDate: timestamp('date').notNull(),
+  clubId: integer('club_id')
```

Comment:
If this is a event specific to a club, name the table `club_events`

Reply:
An event may or may not be related to a club, such as a convocation

Reply:
Then make the `club_id` nullable.